website/docs/misc/contributing.md

```diff
@@ -35,7 +35,7 @@ modify the nav bar) then please refer to the [Docusaurus 2 documentation](https:
 
 ## Previewing changes locally
 
-- Install npm and yarn: see [instructions here](https://v2.docusaurus.io/docs/installation)
```

Comment:
Can we keep this link around too. I think it's useful to point to the docusaurus instructions since they have _many_ more details there.